chain/types/ask.go

```diff
@@ -1,27 +1,9 @@
 package types
 
 import (
-	"github.com/filecoin-project/go-address"
-	cbor "github.com/ipfs/go-ipld-cbor"
+	"github.com/filecoin-project/go-fil-markets/storagemarket"
 )
 
-func init() {
-	cbor.RegisterCborType(SignedStorageAsk{})
-	cbor.RegisterCborType(StorageAsk{})
-}
+type SignedStorageAsk = storagemarket.SignedStorageAsk
 
-type SignedStorageAsk struct {
-	Ask       *StorageAsk
-	Signature *Signature
-}
-
-type StorageAsk struct {
-	// Price per GiB / Epoch
-	Price BigInt
-
-	MinPieceSize uint64
-	Miner        address.Address
-	Timestamp    uint64
-	Expiry       uint64
-	SeqNo        uint64
-}
+type StorageAsk = storagemarket.StorageAsk
```

Comment:
we should make a note to just squash all these aliased types into using the spec ones. Def not needed now, but easy to do over time